core/src/test/java/org/opensearch/sql/expression/datetime/DateSubAndSubDateTest.java

```diff
@@ -29,108 +30,107 @@ private LocalDate today() {
   @Test
   public void subdate_returns_datetime_when_args_are_time_and_time_interval() {
     var res = subdate(LocalTime.of(21, 0), Duration.ofHours(1).plusMinutes(2));
-    assertEquals(DATETIME, res.type());
-    assertEquals(LocalTime.of(19, 58).atDate(today()), res.datetimeValue());
+    assertEquals(TIMESTAMP, res.type());
+    assertEquals(LocalTime.of(19, 58).atDate(today()), res.timestampValue().atZone(UTC_ZONE_ID).toLocalDateTime());
```

Comment:
You can add a helper function in this class to avoid duplicating code

Reply:
added helper function `toLocalDateTime` to reduce duplicated code in https://github.com/Bit-Quill/opensearch-project-sql/commit/f6edc6be62abbe3a3974665e8d51c515584234e0